docs/htdocs/info/docs/tools/vep/script/vep_cache.html

```diff
@@ -194,8 +194,8 @@ <h3 id="cache_content">Data in the cache</h3>
       </tr>
       <tr>
         <td><b>dbSNP</b></td>
-        <td>151</td>
-        <td>151</td>
+        <td>152</td>
+        <td>152</td>
```

Comment:
More changes need to be done, e.g.: 
- COSMIC: version 89 on GRCh38
- RefSeq: 2019-06-28 (GCF_000001405.39_GRCh38.p13_genomic.gff) on GRCh38

Reply:
Thanks!